osi_datarecording.proto

```diff
@@ -27,16 +29,20 @@ message SensorDataList
 {
     // List of sensor data for multiple sensors at a specific timestep.
     //
-    repeated SensorData sensors = 1;
+    // \note OSI uses singular instead of plural for repeated field names.
+    //
+    repeated SensorData sensor = 1;
 }
 
 //
 // \brief List of sensors where each element contains a time series of
-// SensorData messages.
+// \c SensorData messages.
 //
 message SensorDataSeriesList
 {
     // List of sensor data for multiple sensors at subsequent timesteps.
     //
-    repeated SensorDataSeries sensors = 1;
+    // \note OSI uses singular instead of plural for repeated field names.
+    //
+    repeated SensorDataSeries sensor = 1;
```

Comment:
sensor_data_series

The word series is both singular and plural. Same case as "data"

Reply:
Please note that you would adress
sensor_data_series (SensorDataSeriesList) --> sensor_data (SensorDataSeries) -> ... 

often you find in OSI like:
sensor(s) (you get a message SensorDataSeries) -> data (you get a message SensorData) -> ...